### PR TITLE
Feature/assignee

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: app/controllers/api/v1/application_controller.rb [hunk 1]
```diff
@@ -9,5 +9,11 @@ def build_meta_tasks(collection)
       total_pages: collection.total_pages,
       per_page: collection.limit_value
     }
-end
+  end
+
+  def build_meta_users(collection)
```
Comment (artempartos) [inline app/controllers/api/v1/application_controller.rb]: indent

File: app/javascript/packs/components/EditPopup.js [hunk 2]
```diff
@@ -126,9 +135,19 @@ export default class EditPopup extends React.Component {
                 />
               </FormGroup>
             </form>
-            {/*Author: {this.state.author.first_name} {this.state.author.last_name}*/}
+            Author: {this.state.task.author.first_name} {this.state.task.author.last_name}
```
Comment (artempartos) [inline app/javascript/packs/components/EditPopup.js]: use destructing

**Comment by artempartos** [comment]
rebase please

**Comment by moklidia** [comment]
@artempartos  Done. Correctly, I hope)

**Comment by artempartos** [comment]
rebase please, it contains  changes from prev PR

**Comment by moklidia** [comment]
@artempartos  Hm... I rebased this branch to exlude commits from the previous PR and now it says that Pull request is successfully merged and closed... I don't remember closing it))

**Comment by moklidia** [comment]
@artempartos  OMG, looks like something went wrong((

**Pull request closed without merging**